spec/forms/idv/phone_form_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+describe Idv::PhoneForm do
+  let(:user) { build_stubbed(:user, :signed_up) }
+  subject { Idv::PhoneForm.new({}, user) }
+
+  it_behaves_like 'a phone form'
+
+  describe '#submit' do
+    it 'adds phone key to idv_params when valid' do
+      subject.submit(phone: '703-555-1212')
+
+      hash = {
```

Comment:
I would prefer a more descriptive name here such as `expected_param`.   Also, this fits on one line (maybe even inline in the expectation?)